subiquity/common/filesystem/boot.py

```diff
@@ -221,10 +221,11 @@ def get_add_part_plan(device, *, spec, args, resize_partition=None):
 
     create_part_plan = CreatePartPlan(gap=None, spec=spec, args=args)
 
-    if gaps.largest_gap_size(device) >= size:
-        create_part_plan.gap = gaps.largest_gap(device).split(size)[0]
+    gap = gaps.largest_gap(device, in_extended=False)
```

Comment:
I think this function needs a comment referencing that bug that makes logical ESPs fail.

Reply:
Done